### DISCUSSION TITLE
Possibility to set default data source when opening Alerting view

### DISCUSSION THREAD

**Discussion by deimantastumas** [grafana/grafana#53213] [Alerting]
If multiple data sources are present in the organization, containing many alert rules, opening Alerting page becomes super laggy. This is caused due to the fact, that alert rules from all data sources are loaded, and selecting a single data source is only possible post-factum. 

I think that the possibility of setting the default data source on the Alerting page would be quite useful.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
